src/components/Collapsibles.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+
+class Collapsibles extends React.Component {
+  render() {
+    return (
+      <legend className="legend closed">
+        {' '}
```

Comment:
When you saw this `{' '}` is because you have an extra blank space, make sure you don't have any.